spec/forms/password_reset_email_form_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe '#submit' do
     context 'when email is valid and user exists' do
       it 'returns hash with properties about the event and the user' do
-        user = build(:user, :signed_up, email: 'test1@test.com')
+        user = create(:user, :signed_up, email: 'test1@test.com')
```

Comment:
Why does this require creating vs building?

Reply:
The the test is testing the behavior when a user exists. The app does not know the user exists without creating them. This is a bug that was uncovered when changing the personal key behavior. Previously, setting the personal key (which is done in the factory here) was saving the record.

Reply:
Are you sure? This test was working fine before. It knows that the user exists because we are building a user with the same email address as the one that gets passed into the form in the line below this one. If you change the user's email address on line 12, the test will fail.

Reply:
Ah. I see what you mean. The factory was saving the user, so it was acting as `create` instead of `build`. I think the fix is to stub `User.find_with_email` so that it returns a user. We want to minimize accessing the DB in specs. The user doesn't need a personal key in this spec.

Reply:
Anyways, we can do this later since it's not specific to this PR.